src/Protocol/SymbolInformation.php

```diff
@@ -91,6 +90,14 @@ public static function fromNode(Node $node, string $fqn = null)
         }
         if ($node instanceof Node\Name) {
             $symbol->name = (string)$node;
+        } else if(
+            $node instanceof Node\Expr\FuncCall
+            && $node->name instanceof Node\Name
+            && strtolower((string)$node->name) === 'define'
+            && isset($node->args[0])
+            && $node->args[0]->value instanceof Node\Scalar\String_
+        ) {
+            $symbol->name = (string)$node->args[0]->value->value;
```

Comment:
yeah, I see, but there are better options like setting the name early and checking later if the name is already set.